### PR TITLE
Make managing installation URLs easier

### PR DESCRIPTION
**Pull request by stbenjam**
This adds a dropdown menu to the installation page for the OS, and removes all of the other URL's sprinkled throughout the docs.  My javascript skills are not so good, so maybe there's something cleaner to be done.

### CODE REVIEW

**Comment by ehelms** [comment]
I think this looks good (I just noticed the other day Foreman's site doing this).  I've always thought this was a neat idea, and has worked since I remember it -- https://install.openshift.com/

File: docs/installation/index.md [hunk 1]
```diff
@@ -82,43 +72,46 @@ yum-config-manager --disable "*"
...
-yum-config-manager --enable rhel-7-server-extras-rpms
-```
-
-### Enteprise Linux
-
-For all variations of enterprise linux (CentOS and RHEL) the following steps need to be taken:
-
-**EL6**
+<div id="el6" markdown="1">
+**All Enterprise Linux 6 Variants**
```
Comment (ehelms) [inline docs/installation/index.md]: I'd just drop the 'All' here and contemplate dropping the title all together. With the selector displaying exactly my choice, this ends up seeming repetitive.

Comment (stbenjam) [inline docs/installation/index.md]: Indeed, that does look better.  Updated

**Comment by stbenjam** [comment]
@ehelms Yea, RVM does similar. I like it too, maybe we can try it for 2.2?

Is this PR still OK? I'd like to consolidate all the URL's on one page for 2.1

File: docs/installation/index.md [hunk 2]
```diff
@@ -144,3 +136,36 @@ katello-installer <options>
 
 Katello provides a git repository designed to streamline setup by setting up all the proper repositories. Katello deploy provides the ability to deploy a virtual machine instance via Vagrant or direct deployment on an already provisioned machine. For details on how to install using katello-deploy, please see the [README](https://github.com/Katello/katello-deploy/blob/master/README.md).
 
+<script src="/assets/js/jquery.min.js"></script>
```
Comment (ehelms) [inline docs/installation/index.md]: This should already be being included - https://github.com/Katello/katello.org/blob/master/_layouts/base.html#L24

**Comment by ehelms** [comment]
ACK

**Pull request merged**